bam_masterdata/datamodel/object_types.py

```diff
@@ -12694,6 +12694,104 @@ class Crystal(MatSimStructure):
     )
 
 
+class MouseMeasurement(ExperimentalStep):
```

Comment:
I think this should be instead a sub-type of `SaxsMeasurement` right?

Reply:
You can set `show_in_edit_views=False,` for any PropertyTypeAssignment you want to hide. Now that I check this, I have the feeling that the nicer implementation should be: show_in_edit_views is default to True, and then only when marked to False is ignored.

Now, regarding `frame_count`, this is indeed a problem, but for what you say, it seems that there is: one base SaxsMeasurement class, with 2 specific instruments doing this. Translated to classes, this means that you should have another 2 classes, one for each instrument. For now, you can leave it as it is, but for the future we have to be careful with these definitions 🙂